cytominer_eval/tests/test_operations/test_enrichment.py

```diff
@@ -42,37 +42,35 @@
 
 
 def test_enrichment():
-    result = []
-    for p in np.arange(1, 0.97, -0.005):
-        r = enrichment(
-            similarity_melted_df=similarity_melted_df,
-            replicate_groups=replicate_groups,
-            percentile=p,
-        )
-        result.append(r)
-    result_df = pd.DataFrame(result)
+    percent_list = np.arange(1, 0.97, -0.005)
+    result = enrichment(
+        similarity_melted_df=similarity_melted_df,
+        replicate_groups=replicate_groups,
+        percentile=percent_list,
+    )
 
     # check for correct shape and starts with 1.0
-    assert result_df.shape == (7, 4)
-    assert result_df.percentile[0] == 1.0
+    assert result.shape == (7, 4)
+    assert result.enrichment_percentile[0] == 1.0
+    assert result.enrichment_percentile[1] == 0.995
     # check if the higher percentiles are larger than the small one
-    assert result_df.percentile[1] > result_df.percentile.iloc[-1]
+    assert result.enrichment_percentile[1] > result.enrichment_percentile.iloc[-1]
```

Comment:
```suggestion
    assert result.enrichment_percentile[1] > result.enrichment_percentile.iloc[-1]
    
    result_int = enrichment(
        similarity_melted_df=similarity_melted_df,
        replicate_groups=replicate_groups,
        percentile=0.97,
    )
    
    assert result_int.enrichment_percentile[0] == result.enrichment_percentile[1]
```

Would this work? You test precision_recall for both list and int and `evaluate(operation="enrichment")`, but I don't think that you're testing `enrichment` for both.